docs/example.md

```diff
@@ -2,34 +2,47 @@
+The Manifest file (manifest.xml)  informs Tableau about your connector and displays the connector name in the Tableau Connect pane.
 It's a required file that defines the connector class and description.
-The <span style="color:red">class</span> value is a unique key for your connector and is used in other XML files to apply their customizations and in Tableau workbooks to match connection types.
+The <span font-family: courier new">class</span> value is a unique key for your connector and is used in other XML files to apply their customizations and in Tableau workbooks to match connection types.
 
-Each connector is typically based on a "class" such as ODBC or JDBC, and provides additional customizations beyond the class.
-The <span style="color:blue; font-family: courier new">name</span> value displays the connector name in the Tableau **Connect** pane.
+Each connector is typically based on a "class" such as ODBC or JDBC, and provides additional customizations.
 
-You may also specify the vendor information. The <span style="font-family: courier new">company name</span> value displays the name of the connector's creator next to the connector name on the Tableau **Connect** pane (for example, Connector Name by Creator), and the <span style="color:blue; font-family: courier new">support link</span> value is the URL of a website where users of the connector can get support.
+- The <span font-family: courier new">class</span> value is a unique key for your connector and is used in other XML files to apply their customizations and in Tableau workbooks to match connection types.
+- The <span style="font-family: courier new">name</span> value displays the connector name in the Tableau **Connect** pane.
+- You may also specify the vendor information. 
+- The <span style="font-family: courier new">company name</span> value displays the name of the connector's creator next to the connector name on the Tableau **Connect** pane (for example, Connector Name by Creator).
+- The <span style="font-family: courier new">support link</span> value is the URL of a website where users of the connector can get support.
 
 
 ![]({{ site.baseurl }}/assets/manifest-xml.png)
 
 ![]({{ site.baseurl }}/assets/pce-connect-pane.png)
 
-## ![2]({{ site.baseurl }}/assets/pce-2.png) \*.tcd
+## ![2]({{ site.baseurl }}/assets/pce-2.png) 
 
-(Optional) You can use the Tableau Custom Dialog (.tcd) file to customize the connection dialog, or your connector can inherit a dialog from its parent.
-For example, if you include <span style="color:red">show-ssl-check box</span> and set the value to "true", the **Require SSL** check box will display on the sign-in dialog.
+The Tableau Custom Dialog file (.tdc) is optional. By default, your connecor inherits a connection dialog from its parent (defined by <span style="font-family: courier new">superclass</span> You can use the TDC file to customize the connection dialog.
```

Comment:
.tdc -> .tcd
TDC -> TCD

I know, we should go all .xml in the SDK for simplicity.  TFS1072286